package.json

```diff
@@ -39,13 +39,12 @@
     "yargs": "^4.2.0"
   },
   "scripts": {
-    "postinstall": "cd app && npm install",
+    "postinstall": "install-app-deps",
     "build": "gulp build",
     "prerelease": "gulp build --env=production",
     "release": "build --x64 --publish never",
```

Comment:
I didn't touch this line. Why would we need gulp in that place? This should be calling `electron-builder`'s `build`, not gulp :)